docs/en/Help.md

```diff
@@ -1,10 +1,18 @@
 # Ask your local engineers!
 
+## Slack
+
+The COVID-19 Engineering team is on Slack. Anyone with a **canada.ca** email address  can join and participate 
+in the conversation, ask questions etc. at [statcan-aaw.slack.com](https://statcan-aaw.slack.com)
+
 # TODO: Ask Brendan
 
 Feel free to contact...
 
 # To report a bug
 
+Bugs should be reported on the AAW Slack Workspace.  Current issues are logged at 
+[GitHub](https://github.com/statcan/daaas/issues), so please check there before logging a new issue.
+
```

Comment:
I think users should either report it in Slack and not check other sources, or submit the bug directly via GitHub. I lean towards recommending the `#feedback` channel of Slack for the time being and leaving it at that.